scss/bitstyles/objects/button--nav-large/button--nav-large.stories.mdx

```diff
@@ -0,0 +1,85 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Objects/Button--nav-large" />
+
+# Large Nav Button
+
+A button for your main navigation that’s rendered full-width, ideal for the “off-screen-canvas” (or “hamburger”) menu on small screens.
+
+<Canvas>
+  <Story name="Button--nav-large">
+    {`
+      <div class="u-bg--black-80 u-padding-m">
```

Comment:
Plays a bit weird with the light/dark switch on Storybook in my opinion. What's the reason behind giving it a dark background here?

Reply:
Yeh fair enough! I’ve removed the backgrounds (and padding too) on these examples — they should anyway be focused on the component itself, not how to use it. I’ve left the background colors in the navbar examples though, as that’s how it’s intended to appear. (I’ll add a ticket for adding a dark mode though :D)